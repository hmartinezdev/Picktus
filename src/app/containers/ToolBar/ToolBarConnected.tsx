import { signout } from '@store/reducers/user/user-actions';
import { connect } from 'react-redux';
import ToolBar from './ToolBar';

export default connect(
  undefined,
  { signout }
)(ToolBar);
