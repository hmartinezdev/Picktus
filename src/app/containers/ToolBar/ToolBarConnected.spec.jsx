import { shallow } from 'enzyme';
import React from 'react';
import configureStore from 'redux-mock-store';
import ToolBarConnected from './';

let wrapper;
let store;

// create any initial state needed
const initialState = {};

// here it is possible to pass in any middleware if needed into //configureStore
const mockStore = configureStore();

const setup = (props = {}) => shallow(<ToolBarConnected {...props} />);

describe('<ToolBarConnected />', () => {
  beforeEach(() => {
    store = mockStore(initialState);
    wrapper = setup({ store });
  });

  test('should render properly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
