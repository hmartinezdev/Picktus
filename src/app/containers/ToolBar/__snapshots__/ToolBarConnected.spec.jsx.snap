// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ToolBarConnected /> should render properly 1`] = `
<ToolBar
  signout={[Function]}
  store={
    Object {
      "clearActions": [Function],
      "dispatch": [Function],
      "getActions": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
    }
  }
/>
`;
